CHANGELOG.md

```diff
@@ -11,8 +11,9 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 
 ### Added
 
-- We introduced a settings parameters to manage citations' relations local storage time-to-live with a default value set to 30 days. [#11189](https://github.com/JabRef/jabref/issues/11189)
+- We introduced a settings parameter to manage citations' relations local storage time-to-live with a default value set to 30 days. [#11189](https://github.com/JabRef/jabref/issues/11189)
 - We distribute arm64 images for Linux. [#10842](https://github.com/JabRef/jabref/issues/10842)
```

Comment:
Oh, dont we need this news any more?

Reply:
Just fixed a typo, github shows it weird

Reply:
Arm news (lines below the typo fix) was shown deleted here. Now, everything alright!